### ISSUE TITLE
Community Forum agenda point: Accessibility Guide vs Extension of Ethics guide into "Ethics and Inclusion" vs other community suggested option

### ISSUE THREAD

**Issue by malvikasharan** [the-turing-way/the-turing-way#3783]
### Summary

* Our team and community members have been working on accessibility-related content in The Turing Way. 
* We would like to get input from the community members if there should be a new "Accessibility Guide" or current Ethics guide should be extended into "Ethics and Inclusion" or if there are other community suggested option
* We don't currently have a process for adding new guide, so this discussion is a form of creating that path for "Community request and decision on new guides"

### What needs to be done?

- [ ] Collect the pros and cons of these options
- [ ] Open discussion at the community forum
- [ ] Invite comments under this thread
- [ ] Make an informed decision on an option by the end of August/Early September 2024

### Who can help?

* Community
* Accessibility WG
* The project team

---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

**Comment by malvikasharan** [comment]
## Pros and cons of the new accessibility guide:

### Pro: 
- We can get the content created by the community members published in a new guide - we already have a proposal and a few chapters
- Specific accessibility-related collaborations can be established with other open communities/organisations working on this topic
- Accessibility-centred discussions can be visibility held as part of this guide

### Cons
- Maintenance of this guide might fall on accessibility WG that has been primarily been involved in the development of a landing page - that is too much responsibility
- People might see accessibility outside the project design or ethical practice
- We end up creating the guide with a relatively low number of chapters, or the other less developed guides will get further deprioritised

## Another option suggested is extending the guide for Ethics to include accessibility topics (this could be done in project design too?)

### Pro

- We already have a guide in place with few chapters - it can be picked up with new energy brought in by accessibility discussions and content (landing page can be edited with the content developed by the Accessibility WG)
- We have a few community members who are maintaining some parts of the guide - we can have a more shared maintenance by community here

### Con

- Accessibility content will not be findable as easily
-

**Comment by aleesteele** [comment]
Discussed most recently at the @the-turing-way/accessibility-working-group meeting on Monday, 12 August. We'll add more information here after our September meeting!

**Comment by aleesteele** [comment]
_Adding breadcrumbs to #3528_
